### PR TITLE
[ES] Translation of 11-1.md

### PR DESCRIPTION
**Pull request by alceballosa**

### CODE REVIEW

File: docs/es/week11/11-1.md [hunk 1]
```diff
@@ -0,0 +1,1229 @@
+<!-----
+lang-ref: ch.11-1
+lecturer: Yann LeCun
+title: Activation and loss functions (part 1)
+authors: Haochen Wang, Eunkyung An, Ying Jin, Ningyuan Huang
+date: 13 April 2020
+---
+-->
+
```
Comment (Atcold) [inline docs/es/week11/11-1.md:9]: Remove this.
```suggestion

```

File: docs/es/week11/11-1.md [hunk 2]
```diff
@@ -0,0 +1,1229 @@
...
+---
+lang-ref: ch.11-1
+lecturer: Yann LeCun
+title: Activation and loss functions (part 1)
```
Comment (Atcold) [inline docs/es/week11/11-1.md:13]: Title needs a translation.

File: docs/es/week11/11-1.md [hunk 3]
```diff
@@ -0,0 +1,1229 @@
...
+authors: Haochen Wang, Eunkyung An, Ying Jin, Ningyuan Huang
+date: 13 April 2020
+translation-lang: es
```
Comment (Atcold) [inline docs/es/week11/11-1.md:16]:
```suggestion
lang: es
```

File: docs/es/week11/11-1.md [hunk 4]
```diff
@@ -0,0 +1,1229 @@
...
+date: 13 April 2020
+---
+-->
+
+---
+lang-ref: ch.11-1
+lecturer: Yann LeCun
+title: Activation and loss functions (part 1)
+authors: Haochen Wang, Eunkyung An, Ying Jin, Ningyuan Huang
+date: 13 April 2020
```
Comment (Atcold) [inline docs/es/week11/11-1.md:15]:
```suggestion
date: 13 Apr 2020
```

File: docs/es/week11/11-1.md [hunk 5]
```diff
@@ -0,0 +1,1229 @@
...
+translation-lang: es
+translator: alceballosa
+translation-date: 09/10/2020
```
Comment (Atcold) [inline docs/es/week11/11-1.md:18]:
```suggestion
translation-date: 10 Sep 2020
```

File: docs/es/week11/11-1.md [hunk 6]
```diff
@@ -0,0 +1,1229 @@
...
+-->
+
+---
+lang-ref: ch.11-1
+lecturer: Yann LeCun
+title: Activation and loss functions (part 1)
+authors: Haochen Wang, Eunkyung An, Ying Jin, Ningyuan Huang
+date: 13 April 2020
+translation-lang: es
+translator: alceballosa
```
Comment (Atcold) [inline docs/es/week11/11-1.md:17]: You may want to use your real name.

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to update `docs/_config.yml` as well.

**Comment by alceballosa** [comment]
Did all the changes you suggested, thank you professor!

**Review by Atcold — APPROVED** [review]
All is good now. I've updated `/docs/_config.yml` for you this time.

**Pull request merged**